app/src/main/java/com/hisham/ctintegrationsample/home/HomeFragment.kt

```diff
@@ -39,13 +40,20 @@ class HomeFragment : BaseFragment() {
         val countryISO = localStorage.country
         val environment = localStorage.environment
 
+        val landingStyleType = if (localStorage.landingStyle == USPDisplayType.CHECK_STYLE.name) {
```

Comment:
To make it easy to maintain when more USP options comes we could create a constant Map<String, String> to keep this. And instead of if/else we just need to pass the key to get a value